python/gherkin/token_matcher_markdown.py

```diff
@@ -25,6 +25,7 @@ def match_FeatureLine(self, token):
 
         if(self.matched_feature_line):
             self._set_token_matched(token,None)
+            return False
```

Comment:
Note based on CONTRIBUTING.md:

>TL;DR anyone who only knows one of the supported programming languages should be
able to fix a bug or add a feature in all the other implementations. - By virtue of
finding their way around a consistently organised codebase.

There is a difference in behaviour for `_set_token_matched` between the JavaScript and Python implementations, one (javascript) returns true/false based on whether the token was matched (properly?), the other (python) returns `None`

This was a cause for confusion for me, and a source of some of the issues here.